packages/gatsby/src/utils/css-modules-config.js

```diff
@@ -0,0 +1,17 @@
+export function isProduction() {
+  return process.env.NODE_ENV === `production`
+}
+
+export function getLocalIdentName() {
+  return isProduction()
+    ? `[hash:base64]` // Webpack default if unspecified
```

Comment:
see https://github.com/webpack-contrib/css-loader/issues/406 its actually more efficient to use the name with the path and name and local with a sort hash

Reply:
Good to know! Would you recommend sticking with `[path]---[name]---[local]---[hash:base64:5]` for both dev and prod then?

Reply:
yup!